File: MyHordesOptimizerApi/MyHordesOptimizerApi/Dtos/MyHordesOptimizer/Cadavers/CleanUpDto.cs
namespace MyHordesOptimizerApi.Dtos.MyHordesOptimizer
{
    public class CleanUpDto
    {
        public int IdCleanUp { get; set; }
        public CitizenDto CitizenCleanUp { get; set; }
        public CleanUpTypeDto Type { get; set; }
    }
}

File: MyHordesOptimizerWebsite/src/app/_abstract_model/dto/clean-up.dto.ts
import { CitizenDTO } from './citizen.dto';

export interface CleanUpTypeDTO {
    id: number;
    name?: string;
    myHordesApiName?: string;
}

export interface CleanUpDTO {
    idCleanUp: number;
    citizenCleanUp?: CitizenDTO;
    type?: CleanUpTypeDTO;
}
